client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -811,11 +811,9 @@ page-header, .page-header--new {
 // Forms
 .form-control {
   border-radius: @redash-input-radius;
-  color: #9E9E9E;
```

Comment:
Don't we need to use `@input-color` here?

Reply:
Bootstrap has the same class style with `color: @input-color` already, so there's no need to set the color again.

Reply:
Got it. 👍